src/engagement_db_to_analysis/code_imputation_functions.py

```diff
@@ -74,12 +81,25 @@ def _impute_age_category(user, messages_traced_data, analysis_dataset_configs):
                 Metadata(user, Metadata.get_call_location(), TimeUtils.utc_now_as_iso_string())
             )
 
-            messages_traced_data[message_index] = message
+            age_messages +=1
+            imputed_labels +=1
+
+    log.info(f"Imputed {imputed_labels} age category labels for {age_messages} age messages")
 
-        return messages_traced_data
 
 def impute_codes_by_message(user, messages_traced_data, analysis_dataset_config):
```

Comment:
I think this should be analysis_dataset_configs plural? That's what the docstring says...

Reply:
updated thanks.